.gitignore

```diff
@@ -5,6 +5,7 @@ resources/
 
 # npm assets
 node_modules/
+content-modules/
```

Comment:
We need the `content-modules` under version control. I'm curious: is there a reason why you're wanting to ignore them?

Reply:
Probably best to just drop the changes to the file (we don't need the distraction of an extra blank line either). Thanks!